httpstan/views.py

```diff
@@ -634,3 +634,81 @@ async def handle_log_prob(request: aiohttp.web.Request) -> aiohttp.web.Response:
         logger.critical(message)
         return aiohttp.web.json_response(_make_error(message, status=status), status=status)
     return aiohttp.web.json_response({"log_prob": lp}, status=200)
+
+
+async def handle_log_prob_grad(request: aiohttp.web.Request) -> aiohttp.web.Response:
+    """Calculate the gradient.
+
+    ---
+    post:
+      summary: Return the gradient of the unconstrained parameters.
```

Comment:
Is this the right language? Is it the gradient of log_prob evaluated at the provided parameters?